modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,164 +169,165 @@ $ ls $HOME/clusterconfig/
 === Retaining existing partitions
-For an ISO installation, you can add options to the `coreos-installer`
+For an ISO installation, you can add options to the `coreos-installer` command line
 that causes the installer to maintain one or more existing partitions.
-For a PXE installation, you can APPEND coreos.inst options to preserve partitions.
+For a PXE installation, you can `APPEND` `coreos.inst` options to preserve partitions.
 
 Saved partitions might be partitions from an existing {product-title}
-system that has data partitions that you want to keep or partitions
-that you just manually created. Here are a few tips:
+system that has data partitions that you want to keep. Here are a few tips:
 
-* Make sure you assign at least the recommended amount of disk space to the
-OpenShift partitions at the beginning of the disk.
+* If you save existing partitions, and those partitions do not leave enough space for {op-system}, installation will fail without damaging the saved partitions.
 
 * Identify the disk partitions you want to keep either by partition
-number or label.
+label or by number.
 
-For an ISO installation:
+.For an ISO installation
 
-The following example illustrates running the coreos-installer in a way that preserves
-the sixth (6) partition on the disk:
+This example preserves any partition in which the partition label begins with `data` (`data*`):
 
 [source,terminal]
 ----
 # coreos-installer install --ignition-url http://10.0.2.2:8080/user.ign \
-        --save-partindex 6 /dev/sda
+        --save-partlabel 'data*' /dev/sda
 ----
 
-This example preserves partitions 5 and higher:
+The following example illustrates running the `coreos-installer` in a way that preserves
+the sixth (6) partition on the disk:
 
 [source,terminal]
 ----
-# coreos-installer install --ignition-url http://10.0.2.2:8080/user.ign
-        --save-partindex 5- /dev/sda
+# coreos-installer install --ignition-url http://10.0.2.2:8080/user.ign \
+        --save-partindex 6 /dev/sda
 ----
 
-This example preserves any partition in which the partition label begins with `data` (`data*`):
+This example preserves partitions 5 and higher:
 
 [source,terminal]
 ----
 # coreos-installer install --ignition-url http://10.0.2.2:8080/user.ign
-        --save-partlabel ‘data*’ /dev/sda
+        --save-partindex 5- /dev/sda
 ----
 
-In all of those examples, after the install completes, on the first boot of
-the new system, Ignition will recreate the partition at the same place it
-was before (the same offset), and the filesystem will be intact.
+In the previous examples where partition saving is used, `coreos-installer` recreates the partition immediately.
 
-For a PXE installation:
+.For a PXE installation
 
-This APPEND option preserves the sixth partition:
+This `APPEND` option preserves any partition in which the partition label begins with 'data' ('data*'):
 
 [source,terminal]
 ----
-coreos.inst.save_part=6
+coreos.inst.save_partlabel='data*'
```

Comment:
For the `save_partlabel` kernel argument, there are no quotes; this should just be:

```
coreos.inst.save_partlabel=data*
```